config/config.yml.example

```diff
@@ -126,6 +126,11 @@ base: &base
     # same code running at CONFIG.tile_servers.elasticsearch
     node_api_url: http://localhost:4000/v1
 
+    # The secrets used to sign the JWT for authenticated requests to the
+    # iNaturalistAPI. These need to be same as in the API's config.js.
+    jwt_secret: jwtSecret
+    jwt_application_secret: jwtApplicationSecret
```

Comment:
I noticed a few places in the config where a fallback is configured:

```ruby
CONFIG.jwt_secret || "secret"
...
CONFIG.jwt_application_secret || "application_secret"
```
But the fallback in js is slightly different.

Maybe the example configs and the runtime fallbacks could be made more consistent overall, but I like where it's headed.

Reply:
Ah yes I hadn't noticed the fallbacks were different on each side for the application secret at least. 
It would also be nice to make the example configs match the fallbacks too.
Would this change be worthy of a new issue?

Reply:
IMO, this is equally trivial, so if you want to put it in a PR we can just review it there.